### DISCUSSION TITLE
Menus / Submenus

### DISCUSSION THREAD

**Discussion by Kx29PM** [btcpayserver/btcpayserver#4597] [Ideas / Feature Requests]
Hi guys, we would really appreciate the option of putting articles on separate menus as we run a cafe and just a list of 90 items is impossible to use. 

This is how our General Bytes PoS looks in and in the backend. 

![menus](https://user-images.githubusercontent.com/124400297/216705331-eb7f7aaf-eb73-4f42-a726-a5f5aad858b5.jpeg)

<img width="1276" alt="categories" src="https://user-images.githubusercontent.com/124400297/216705327-e3ce4e90-918b-4a36-a523-edea33094bbc.png">

**Comment by pavlenex** [comment]
We have categories https://blog.btcpayserver.org/btcpay-server-1-12-0/#%EF%B8%8F-point-of-sale-improvements
